CHANGELOG.md

```diff
@@ -29,6 +29,13 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   that describes them better, and they also behave similar to Mercurial's
   bookmarks.
 
+* `jj unsquash` has been deprecated in favor of `jj squash`.
+
+  **Rationale:** `jj squash` can do everything `jj unsquash` does, and more.
```

Comment:
This isn't quite true. Copying from Discord:

>  [...] if your parent commit changes `println!("foo")` to `println!("bar")` and your child commit changes it again to `println!("baz");`, then you can use `jj unsquash` to add the missing semicolon to the parent commit (i think - i'm not sure i've ever used the command myself).

Note that you can't test that with scm-record because it doesn't have a way of letting you edit the right.

Reply:
> This isn't quite true. Copying from Discord:
> […]

I'll add a PR for `diffedit --preserve-descendants` before reproposing this one.

Reply:
Done.